### PR TITLE
update convert_mesh.py: new '23d' option, write shell cells only

### PR DESCRIPTION
**Pull request by vlukes**

### CODE REVIEW

**Comment by rc** [comment]
What about adding instead a `--cell-dim` option, that could take a list of ints from {1, 2, 3}? This could replace the `-23` as well as `-3`. The `-2` is different and would stay (it is used for fixing planar files with zero/constant z coordinate).

So:
`-3` -> `--cell-dim=3`
`-23` -> `--cell-dim=2`

I would allow also  `--cell-dim=2,3` for meshes to be used outside sfepy.

**Comment by vlukes** [comment]
OK, `--cell-dim` (short version is  `-d`) option added. Also syntax `--cell-dim=2,3`is allowed.

File: sfepy/discrete/fem/mesh.py [hunk 1]
```diff
@@ -386,7 +386,11 @@ def _get_io_data(self, cell_dim_only=None):
         cmesh = self.cmesh
         conns, mat_ids = [], []
         if cell_dim_only is not None:
-            descs = [ii for ii in self.descs if int(ii[0]) == cell_dim_only]
+            try:
+                iter(cell_dim_only)
```
Comment (rc) [inline sfepy/discrete/fem/mesh.py]: Maybe use `is_sequence()` from sfepy.base.base

File: script/convert_mesh.py [hunk 2]
```diff
@@ -39,7 +39,7 @@
     'tri-tetra' : 'convert elements: quad->tri, hexa->tetra',
     '2d' : 'force a 2D mesh by removing the z coordinates - assumes a 3D mesh'
     ' in the xy plane',
-    '3d' : 'force a 3D mesh by removing 1D and 2D cells',
+    'cell-dim' : 'write only cells of a given dimension(s)',
```
Comment (rc) [inline script/convert_mesh.py]: Maybe more explicitly:
"write only cells of a given dimension - use a comma-separated list for several values."

**Comment by rc** [comment]
Thanks! I have two small suggestions, otherwise LGTM.

File: script/convert_mesh.py (hunk 2 above)
Comment (vlukes) [reply script/convert_mesh.py]: updated

**Comment by rc** [comment]
`is_sequence()` does not work?

File: sfepy/discrete/fem/mesh.py (hunk 1 above)
Comment (vlukes) [reply sfepy/discrete/fem/mesh.py]: It works but I'm not so fast.

**Comment by rc** [comment]
Thanks, merging!

**Pull request merged**